okapi-core/pom.xml

```diff
@@ -66,11 +71,6 @@
       <groupId>io.vertx</groupId>
       <artifactId>vertx-pg-client</artifactId>
     </dependency>
-    <dependency>  <!-- needed because OpenJDK 8 doesn't support TLSv1.3 -->
```

Comment:
@julianladisch is the removal of boringssl related to the vert.x upgrade? If not, maybe it would be better to do it in a separate PR?